docs/brand-guide.rst

```diff
@@ -64,6 +64,8 @@ Under the following terms:
 
 When using the Solidity logo, please respect the Solidity logo guidelines.
 
+You can download the logo `here <https://github.com/ethereum/solidity/blob/develop/docs/logo.svg>`_.
```

Comment:
Not sure about blind people specifically but I think that having the link title say what it links too is good for accessibility too.

```suggestion
You can download `logo.svg <https://github.com/ethereum/solidity/blob/develop/docs/logo.svg>`_ from our repository.
```

Reply:
I think the link is actually redundant since the logo is right below.

Reply:
I think it's not since the svg version should be linked somewhere. :)

Reply:
It is the svg:
![Screenshot](https://user-images.githubusercontent.com/20340/93449000-1fdfd100-f8cc-11ea-9628-ffd01f092ba2.png)

Reply:
If you all think it's enough fair enough... I think it doesn't hurt to make it more explicit.

Reply:
We are trying to limit the links to the github repository to avoid dead links in the future (given each documentation version is archived).

I guess one alternative could be saying right click to download the logo?

Reply:
Good idea, I've adjusted it!